src/components/Calculator.js

```diff
@@ -2,50 +2,48 @@
 import React, { Component } from 'react';
 // eslint-disable-next-line
 import calculate from './logic/calculate';
+// eslint-disable-next-line
```

Comment:
- [ ] It is not allowed to disable any linter rules in all your files. you are expected to work towards solving those syntax errors instead of disabling linter rules because by doing so, you are making a room for bugs in the future. Please remove all the codes concerning disabling linter rules in all your files (including App.js) 😃 👍